pkg/js/helpers.js

```diff
@@ -156,6 +170,18 @@ function ALIAS(name, target) {
     }
 }
 
+// CAA(name,tag,value, recordModifiers...)
+function CAA(name, tag, value){
+    checkArgs([_.isString, _.isString, _.isString], arguments, "CAA expects (name, tag, value)")
```

Comment:
I certainly don't want people to have to do binary math. :-)

Let's keep it as-is.

    CAA("@", "tag", "target", {meta: "whatever"}, CAA_CRITICAL, {more_meta:"foo")

The json should show an integer, not CAA_CRITICAL.  That way we can add more flags (CAA_FUTURE_USE) easily.